x-pack/test/accessibility/apps/ml.ts

```diff
@@ -0,0 +1,351 @@
+          await ml.singleMetricViewer.selectEntityValue('airline', 'AAL');
+
+          await a11y.testAppSnapshot();
+        });
+
+        it.skip('anomaly detection Anomaly Explorer page', async () => {
+          // Skip test until the dots used in the Elastic chart legend no longer have duplicate ids
+          // see https://github.com/elastic/elastic-charts/issues/970
+          await ml.singleMetricViewer.openAnomalyExplorer();
+          await ml.commonUI.waitForMlLoadingIndicatorToDisappear();
+          await a11y.testAppSnapshot();
+        });
+
+        it('data frame analytics page', async () => {
+          await ml.navigation.navigateToDataFrameAnalytics();
+          await a11y.testAppSnapshot();
+        });
+
+        it('data frame analytics outlier job exploration page', async () => {
+          await ml.dataFrameAnalyticsTable.openResultsView(dfaOutlierJobId);
+          await ml.dataFrameAnalyticsResults.assertOutlierTablePanelExists();
+          await ml.dataFrameAnalyticsResults.assertResultsTableExists();
+          await ml.dataFrameAnalyticsResults.assertResultsTableNotEmpty();
+          await a11y.testAppSnapshot();
+        });
+
+        it('data frame analytics create job select index pattern modal', async () => {
+          await ml.navigation.navigateToDataFrameAnalytics();
+          await ml.dataFrameAnalytics.startAnalyticsCreation();
+          await a11y.testAppSnapshot();
+        });
+
+        it('data frame analytics create job configuration step', async () => {
+          await ml.testExecution.logTestStep(
+            'job creation selects the source data and loads the DFA job wizard page'
+          );
+          await ml.jobSourceSelection.selectSourceForAnalyticsJob(ihpIndexPattern);
+          await ml.dataFrameAnalyticsCreation.assertConfigurationStepActive();
+          await a11y.testAppSnapshot();
+        });
+
+        it('data frame analytics create job configuration step for outlier job', async () => {
+          await ml.testExecution.logTestStep('selects the outlier job type');
+          await ml.dataFrameAnalyticsCreation.assertJobTypeSelectExists();
+          await ml.dataFrameAnalyticsCreation.selectJobType(dfaJobType);
+          await ml.testExecution.logTestStep('displays the source data preview');
+          await ml.dataFrameAnalyticsCreation.assertSourceDataPreviewExists();
+          await ml.testExecution.logTestStep('enables the source data preview histogram charts');
+          await ml.dataFrameAnalyticsCreation.enableSourceDataPreviewHistogramCharts();
+          await a11y.testAppSnapshot();
+        });
+
+        it('data frame analytics create job additional options step', async () => {
+          await ml.dataFrameAnalyticsCreation.continueToAdditionalOptionsStep();
+          await a11y.testAppSnapshot();
+        });
+
+        it('data frame analytics create job additional options step', async () => {
```

Comment:
See above: do we want to add a `for outlier job` to the test title?

Reply:
Done in https://github.com/elastic/kibana/pull/88323/commits/55ed7968b20327dd9d5205bfb6c89d877aa3fd92